tests/integration/tier0/check-internet-connection/test_internet_connection.py

```diff
@@ -42,17 +42,19 @@ def _restore_configuration_files():
         shutil.copy(RESOLV_CONF_BACKUP_FILE, RESOLV_CONF_FILE)
 
 
+@pytest.mark.available_connection
 def test_check_if_internet_connection_is_reachable(convert2rhel):
     """Test if convert2rhel can access the internet."""
-    with convert2rhel(("--no-rpm-va --debug")) as c2r:
+    with convert2rhel("--no-rpm-va --debug") as c2r:
         c2r.expect("Checking internet connectivity using address")
-        assert c2r.expect_exact("Internet connection available.") == 0
+        assert c2r.expect_exact("internet connection seems to be available") == 0
```

Comment:
I have modified the expected string, the previous could not be read, as the log message is defined differently.